test/extension/test.sh

```diff
@@ -227,6 +227,7 @@ function tear_down() {
     sql 'DROP ROLE cdb_testmember_2;'
 
     tear_down_database
+    DATABASE=postgres sql postgres 'DROP ROLE IF EXISTS publicuser';
```

Comment:
it can fail, but as stated in the readme:
https://github.com/CartoDB/cartodb-postgresql#test-installation
> NOTE: the tests need to run against a clean postgres instance, if you have some roles already created test will likely fail due publicuser not being dropped.